tests/pass/available-parallelism-with-isolation.rs

```diff
@@ -0,0 +1,8 @@
+//@compile-flags: -Zmiri-isolation-error=hide -Zmiri-num-cpus=1024
```

Comment:
The filename makes no sense, since the existing available-parallelism test already has isolation enabled. Also you don't need isolation-error=hide any more.